public/wp-content/themes/glific/page-features.php

```diff
@@ -33,11 +33,11 @@
 					<?php foreach ($demo_videos['videos'] as $key => $video) :
 						$video_thumbnail = get_thumbnail_from_youtube_video($video['video']);
 					?>
-					<div class="d-flex flex-column mx-4 mx-md-0 w-124 w-xl-214 mb-md-6 mb-xl-13 single-video c-pointer position-relative" data-target="<?php echo "#video-$key" ; ?>">
-						<img src="<?php echo $video_thumbnail ; ?>" class="w-124 h-80 h-xl-136 w-xl-full border-0 rounded-15">
+					<div class="d-block h-auto flex-column mx-4 mx-md-0 w-124 w-xl-214 mb-md-6 mb-xl-13 single-video c-pointer position-relative h-full" data-target="<?php echo "#video-$key" ; ?>">
+						<img src="<?php echo $video_thumbnail ; ?>" class="w-124 h-80 h-xl-136 w-xl-full border-0 rounded-15 d-block">
 						<h5 class="font-heebo-bold fz-12 leading-18 mt-4 mb-0"><?php echo $video['name']; ?></h5>
 						<p class="font-heebo-regular fz-12 leading-18 mb-0 mt-xl-7 text-left"><?php echo $video['time']; ?></p>
-						<div class="w-20 w-xl-35 position-absolute mt-9 mt-xl-15.5 ml-15 ml-xl-24">
+						 <div class="w-20 w-xl-35 position-absolute ml-15 ml-xl-24 top-8.5 top-xl-16">
```

Comment:
unnecessary change in formating. Could you format the whole file before merging?